stubs/setuptools/setuptools/msvc.pyi

```diff
@@ -0,0 +1,176 @@
+from typing import ClassVar, Final, TypedDict, overload, type_check_only
+from typing_extensions import LiteralString, NotRequired
+
+class winreg:
+    HKEY_USERS: Final[int | None]
+    HKEY_CURRENT_USER: Final[int | None]
+    HKEY_LOCAL_MACHINE: Final[int | None]
+    HKEY_CLASSES_ROOT: Final[int | None]
+
+environ: dict[str, str]
+
+class PlatformInfo:
+    current_cpu: ClassVar[str]
```

Comment:
Maybe `Final` would be more appropriate?

Reply:
Agreed, `PlatformInfo` looks like some readonly infromation class based on the architecture.

Reply:
Agreed, `PlatformInfo` looks like some readonly infromation class based on the architecture.
```suggestion
    curren_cpu: Final[str]
```